### PR TITLE
Created a ComponentListTemplate extension to enable sorting

### PR DESCRIPTION
**Pull request by TRoZZNL**
- Sorting is re-enabled within an extension method.
- Fixed the JS sorting button bug, which would fire twice and show enabled -> disabled .
- Removed Slip package dependancy from the Grunt Bundle.

@marcmolenwijk a new CDN package should be created for the Bundle.

### CODE REVIEW

File: src/Sushi.Mediakiwi.Files/gruntfile.js [hunk 1]
```diff
@@ -87,7 +87,7 @@ module.exports = function (grunt) {
                             'compiled/jquery-ambiance.min.js',
                             'compiled/fixedBar.min.js',
                             'compiled/jquery-nicescroll.min.js',
-                            'compiled/slip.min.js',
+                            /*'compiled/slip.min.js',*/
```
Comment (bart-van-der-drift) [inline src/Sushi.Mediakiwi.Files/gruntfile.js:90]: Please remove commented out code

File: src/Sushi.Mediakiwi.Files/gruntfile.js [hunk 2]
```diff
@@ -109,7 +109,7 @@ module.exports = function (grunt) {
                             'compiled/jquery-ambiance.min.js',
                             'compiled/fixedBar.min.js',
                             'compiled/jquery-nicescroll.min.js',
-                            'compiled/slip.min.js',
+                           /* 'compiled/slip.min.js',*/
```
Comment (bart-van-der-drift) [inline src/Sushi.Mediakiwi.Files/gruntfile.js:112]: Please remove commented out code

File: src/Sushi.Mediakiwi.Files/responsive/list.shtml [hunk 3]
```diff
@@ -31,7 +31,7 @@
 	<script type="text/javascript" src="//az609478.vo.msecnd.net/6-25/scripts/jquery-hoverIntent.js"></script>
 	<script type="text/javascript" src="//az609478.vo.msecnd.net/6-25/scripts/jquery-curtainMenu.js"></script>
 	<script type="text/javascript" src="//az609478.vo.msecnd.net/6-25/scripts/jquery-ambiance.js"></script>
-	<script type="text/javascript" src="//az609478.vo.msecnd.net/6-25/scripts/slip.js"></script>
+	<!--<script type="text/javascript" src="//az609478.vo.msecnd.net/6-25/scripts/slip.js"></script>-->
```
Comment (bart-van-der-drift) [inline src/Sushi.Mediakiwi.Files/responsive/list.shtml:34]: Please remove commented out code

File: src/Sushi.Mediakiwi.Files/responsive/test.shtml [hunk 4]
```diff
@@ -31,7 +31,7 @@
 	<script type="text/javascript" src="//az609478.vo.msecnd.net/6-25/scripts/jquery-hoverIntent.js"></script>
 	<script type="text/javascript" src="//az609478.vo.msecnd.net/6-25/scripts/jquery-curtainMenu.js"></script>
 	<script type="text/javascript" src="//az609478.vo.msecnd.net/6-25/scripts/jquery-ambiance.js"></script>
-	<script type="text/javascript" src="//az609478.vo.msecnd.net/6-25/scripts/slip.js"></script>
+	<!--<script type="text/javascript" src="//az609478.vo.msecnd.net/6-25/scripts/slip.js"></script>-->
```
Comment (bart-van-der-drift) [inline src/Sushi.Mediakiwi.Files/responsive/test.shtml:34]: Please remove commented out code

**Review by bart-van-der-drift — CHANGES_REQUESTED** [review]
Please see comments

File: src/Sushi.Mediakiwi.Files/gruntfile.js (hunk 1 above)
Comment (TRoZZNL) [reply src/Sushi.Mediakiwi.Files/gruntfile.js]: Done

File: src/Sushi.Mediakiwi.Files/responsive/list.shtml (hunk 3 above)
Comment (TRoZZNL) [reply src/Sushi.Mediakiwi.Files/responsive/list.shtml]: Done

File: src/Sushi.Mediakiwi.Files/responsive/test.shtml (hunk 4 above)
Comment (TRoZZNL) [reply src/Sushi.Mediakiwi.Files/responsive/test.shtml]: Done

**Review by bart-van-der-drift — APPROVED** [review]

**Pull request merged**